src/main/kotlin/no/nav/syfo/client/DokArkivClient.kt

```diff
@@ -134,3 +136,8 @@ fun Behandler.formatName(): String =
         } else {
             "$etternavn $fornavn $mellomnavn"
         }
+
+fun norskFormatDato(dato: LocalDate): String {
```

Comment:
Jeg ville sikkert bare kalt metoden for "formatDate()" eller noe, men det er litt smak og behag :)